storage/headers.go

```diff
@@ -24,4 +24,10 @@ type Headers interface {
 	// be unfinalized; if there is more than one, at least one of them has to
 	// be unfinalized.
 	ByParentID(parentID flow.Identifier) ([]*flow.Header, error)
+
+	// Indexes block ID by chunk ID
+	IndexByChunkID(headerID, chunkID flow.Identifier) error
+
+	// Finds the ID of the block containing given collection
+	IDByChunkID(chunkID flow.Identifier) (flow.Identifier, error)
```

Comment:
I would name this method as `BlockID(chunkID flow.Identifier) ....`.

Reply:
Also, godoc may need refactoring (collection?)

Reply:
But we return only Identifier not whole object, name like `BlockID` would be misleading.
Fixed the parameter name though